<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hyperspectral viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../artwork/Logo.png</normaloff>../artwork/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="batchSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Bands</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="channelLabel">
      <property name="text">
       <string>Channel (c)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="channelSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QGraphicsView" name="imageCanvas">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="batchLabel">
      <property name="text">
       <string>Batch index (n)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxColors"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExport_as_PNG"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Channel order</string>
     </property>
     <addaction name="order_H_W_C"/>
     <addaction name="order_C_H_W"/>
     <addaction name="order_W_C_H"/>
    </widget>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color (3 channels)</string>
     </property>
     <addaction name="color_RGB"/>
     <addaction name="color_BGR"/>
     <addaction name="color_GBR"/>
    </widget>
    <widget class="QMenu" name="menuColor_2">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="color_Grayscale"/>
     <addaction name="color_Colormap"/>
    </widget>
    <addaction name="menuMode"/>
    <addaction name="actionSet_Workspace"/>
    <addaction name="menuColor_2"/>
    <addaction name="menuColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionExport_as_PNG"/>
   <addaction name="separator"/>
   <addaction name="actionHistogram"/>
   <addaction name="separator"/>
   <addaction name="color_Grayscale"/>
   <addaction name="color_Colormap"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="color_RGB"/>
   <addaction name="color_BGR"/>
   <addaction name="color_GBR"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/artwork/document-open.svg</normaloff>:/artwork/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open numpy file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_as_PNG">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/artwork/document-save.svg</normaloff>:/artwork/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Export as PNG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/histogram.png</normaloff>../artwork/histogram.png</iconset>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="order_C_H_W">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIP (N*C*H*W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="order_H_W_C">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BSQ (N*H*W*C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="color_Colormap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/icolormap.png</normaloff>../artwork/icolormap.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="color_RGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/rgb.png</normaloff>../artwork/rgb.png</iconset>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="color_BGR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/bgr.png</normaloff>../artwork/bgr.png</iconset>
   </property>
   <property name="text">
    <string>BGR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="color_Grayscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/gray.png</normaloff>../artwork/gray.png</iconset>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="order_W_C_H">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIL (N*W*C*H)</string>
   </property>
  </action>
  <action name="color_GBR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/gbr-removebg-preview.png</normaloff>../artwork/gbr-removebg-preview.png</iconset>
   </property>
   <property name="text">
    <string>GBR</string>
   </property>
  </action>
  <action name="actionSet_Workspace">
   <property name="text">
    <string>Set Workspace</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
